docs/docs/publish-azure-devops.md

```diff
@@ -0,0 +1,100 @@
+# Setting up DocFX with Azure DevOps to Deploy to GitHub Pages
```

Comment:
Do we also want this to appear in the [Table of Content](https://github.com/dotnet/docfx/blob/main/docs/docs/toc.yml)

Reply:
I think so. Let me know if that needs to be done, and I will update it.

Reply:
Yes please